src/components/Collapsable.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import { Box, Button, Collapsible, Grommet, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const MenuButton = ({ label, open, submenu, ...rest }) => {
+  return (
+    <Button hoverIndicator {...rest}>
+      <Box
+        background="dark-3"
+        margin={submenu ? { left: 'small' } : undefined}
+        direction="row"
+        align="center"
+        pad={{
+          horizontal: 'medium',
+          vertical: 'small'
+        }}
+      >
+        <Text>{label}</Text>
+      </Box>
+    </Button>
+  );
+};
+
+export default function CollapsibleMenu() {
+  const [openMenu, setOpenMenu] = useState(false);
+  function toggleMenu() {
+    setOpenMenu(!openMenu);
+  }
+
+  return (
+    <Grommet theme={grommet}>
+      <Box width="small">
+        <MenuButton open={openMenu} label="Decadevs" onClick={toggleMenu} />
+        {['All Decadevs', 'Decadevs Interviewed', 'Decadevs Hired'].map(
```

Comment:
create a folder called constants and a file in it; move this array into the file, import it here and use it